packages/back-end/src/app.ts

```diff
@@ -150,6 +152,26 @@ app.get("/favicon.ico", (req, res) => {
   res.status(404).send("");
 });
 
+let robotsTxt = "";
+app.get("/robots.txt", (_req, res) => {
+  if (!robotsTxt) {
+    const file =
+      process.env.ROBOTS_TXT_PATH || path.join(__dirname, "..", "robots.txt");
```

Comment:
do we want a default  .robots.txt in out repo?

Reply:
Maybe? I didn't feel like I knew the tradeoffs there well enough so I opted not to create it, just surface the route so customers can add it if they want to.

Reply:
I like it this way. Meaningful default with and overridable option. Would be nice to document this somwehere as well.

Reply:
Yeah this definitely deserves docs. Was gonna toss that to Ryan after I merge this cause I still don't have a great mental map of what docs exist already/where to add it

Reply:
They're in `/docs` in the repo! It would be nice to add it to the PR so that it self-contained.